CLKernel.cpp

```diff
@@ -391,6 +392,22 @@ void CLKernel::run(cl_command_queue *queue, int ND, const size_t *global_ws, con
     nextArg = 0;
 }
 
+int CLKernel::getPreferredWorkGroupSizeMultiple()
+{
+    if (preferredMultiple != -1)
```

Comment:
this function seems fine: doesnt modify any existing behavior, and potentailly useful => cool :-)